### PR TITLE
Add steps to rollback to kube-dns

### PR DESCRIPTION
**Pull request by rajansandeep**
Adds instructions to rollback to kube-dns for clusters installed with CoreDNS.

### CODE REVIEW

**Comment by rajansandeep** [comment]
fyi @chrisohaver @fturib

**Comment by Bradamant3** [comment]
/hold
/assign

@kubernetes/sig-cluster-lifecycle-pr-reviews PTAL

File: content/en/docs/tasks/administer-cluster/coredns.md [hunk 1]
```diff
@@ -58,6 +65,23 @@ customizations after the new ConfigMap is up and running.
...
+
+```
+kubeadm upgrade apply <current-installed-version> --feature-gates CoreDNS=false --force
+```
+This will replace CoreDNS with kube-dns.
+
+Alternately, you can install kube-dns manually with the help of these [instructions](https://github.com/coredns/deployment/tree/master/kubernetes#rollback-to-kube-dns).
+
+**Be advised that there is no translation from the CoreDNS configmap to the kube-dns configmap provided.
+If there is an old kube-dns configmap that had been applied, it will remain and not be deleted.**
```
Comment (xiangpengzhao) [inline content/en/docs/tasks/administer-cluster/coredns.md]: So, do users need to delete the old kube-dns configmap manually first?

File: content/en/docs/tasks/administer-cluster/coredns.md [hunk 2]
```diff
@@ -58,6 +65,23 @@ customizations after the new ConfigMap is up and running.
...
+
+
```
Comment (dixudx) [inline content/en/docs/tasks/administer-cluster/coredns.md]: Remove empty lines.

File: content/en/docs/tasks/administer-cluster/coredns.md (hunk 1 above)
Comment (dixudx) [reply content/en/docs/tasks/administer-cluster/coredns.md]: Kube-dns will optionally mount kube-dns configmap if existed. So if the settings in the old kube-dns configmap work, IMO there is no need to manually delete it first.

Comment (xiangpengzhao) [reply content/en/docs/tasks/administer-cluster/coredns.md]: Agreed. But we need to document this case to users :)

Comment (rajansandeep) [reply content/en/docs/tasks/administer-cluster/coredns.md]: Yes, the kube-dns configmap remains undeleted. 
Users can choose to delete it, though it's not needed.

File: content/en/docs/tasks/administer-cluster/coredns.md [hunk 3]
```diff
@@ -58,6 +65,21 @@ customizations after the new ConfigMap is up and running.
 If you are running CoreDNS in Kubernetes version 1.11 and later, during upgrade,
 your existing Corefile will be retained.
 
+## Rollback to kube-dns
+
+From Kubernetes v1.11, Kubeadm will install CoreDNS by default.
+If you want to rollback your cluster to use kube-dns, use the following command:
+
+```
+kubeadm upgrade apply <current-installed-version> --feature-gates CoreDNS=false --force
```
[the remark this answers is not in the document]
Comment (timothysc) [reply content/en/docs/tasks/administer-cluster/coredns.md]: Let's put a warning that this is not recommended.  

Once they do this they are in a weird space for 1.13.

**Comment by rajansandeep** [comment]
@Bradamant3 
is there anything else required to merge this PR?

**Comment by neolit123** [comment]
bump, this would be nice to have as soon as 1.12 is released. :\

**Comment by zparnold** [comment]
I have it on good authority @Bradamant3 will be getting to this tomorrow. 😄

**Comment by johnbelamaric** [comment]
Please hold off merging - CoreDNS-as-default unfortunately is pushed to 1.13 due to a last-minute scalability issue for clusters > ~1000 nodes.

**Comment by johnbelamaric** [comment]
Actually, nevermind, this is discussing kubeadm specifically which is OK.

**Comment by zparnold** [comment]
Per @Bradamant3 's comment:
>So I am contemplating the coredns mess that is PRs 10000 and 10012, and I think what I'd like to do is hold off on choosing either one, wait until after the release, and then drop a new and cleaner PR in (closing both these two, but not until we have a better alternative in place)

/close

**Comment by rajansandeep** [comment]
>So I am contemplating the coredns mess that is PRs 10000 and 10012

@zparnold PR 10012?

**Pull request closed without merging**